docs/network/default_cni_plugin.md

```diff
@@ -149,3 +150,43 @@ The specified interface will be added in OVS bridge `br-ex` which acts as gatewa
 microshift-ovs-init.service is able to setup one additional host interface for cluster ingress/egress traffic.
 This is done by specifying the `externalGatewayInterface` in the CNI config file `/etc/microshift/ovn.yaml`.
 The external gateway interface will be added in a second OVS bridge `br-ex1`. Cluster pod traffic destinated to additional host subnet will be routed through `br-ex1`.
+
+### Blocking external access to NodePort service on specific host interfaces
+
+ovn-kubernetes doesn't restrict the host interfaces where NodePort service can be accessed from outside MicroShift node. The following `nft` instructions block NodePort service on a specific host interface.
+
+<br>
+
+Insert a new rule in table `ip nat` chain `PREROUTING` to drop the packet with matching destination port and ip:
+```text
+(host)$ NODEPORT=30700
+(host)$ INTERFACE_IP=192.168.150.33
+(host)$ nft -a insert rule ip nat PREROUTING tcp dport $NODEPORT ip daddr $INTERFACE_IP drop
+```
+> Replace value of NODEPORT variable with the host port number assigned to kubernetes NodePort service
+> Replace value of INTERFACE_IP with the IP address from the host interface where you'd like to block the NodePort service
```

Comment:
I'm not sure about this, but I looked at "rich diff" and these two lines were merged into one. Maybe adding one empty with `>` will keep them separate but in the same quote block

```suggestion
> Replace value of NODEPORT variable with the host port number assigned to kubernetes NodePort service
>
> Replace value of INTERFACE_IP with the IP address from the host interface where you'd like to block the NodePort service
```

Reply:
added line break at the end of each line.